docs/controls/wpf-winforms/WindowsXamlHost.md

```diff
@@ -15,7 +15,7 @@ You can use the **WindowsXamlHost** control to add any Universal Windows Platfor
 * Any first-party UWP control provided by the Windows SDK or WinUI library.
 * Any custom UWP control. You must have the source code for the custom control so you can compile it with your application.
     > [!IMPORTANT]
-    > Using the [WindowsXamlHost](/windows/communitytoolkit/controls/wpf-winforms/windowsxamlhost) control to host custom UWP controls is supported only in apps that target .NET Core 3. This scenario is not supported in apps that target the .NET Framework.
+    > Using the [WindowsXamlHost]() control to host custom UWP controls is supported only in apps that target .NET Core 3. This scenario is not supported in apps that target the .NET Framework.

```

Comment:
Looks like the link was deleted entirely?

Reply:
It was circular (pointed to the article it was in).  Normally means it should be removed.